docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,56 @@
+# Add conditions to TODO comments to make them error
+
+Makes possible to pass arguments to TODO and FIXME comments to trigger errors.
+
+Currently supporting:
+
+* `[YYYY-MM-DD]` to define a due date.
```

Comment:
What about supporting date ranges (e.g. `[startDate-endDate]`)? It could be useful in some cases.

Reply:
> It could be useful in some cases.

What cases?

Reply:
I'm not sure, but others might come up with some 😅

Anyways, the rule looks good to me overall (aside for the suggestions you made above and other minor implementation improvements).

Reply:
I hope you don't mind I mark this a solved.